panther_analysis_tool/backend/graphql/sdk_upload.graphql

```diff
@@ -0,0 +1,9 @@
+mutation SDKUpload($input: UploadDetectionEntitiesInput!) {
+    uploadDetectionEntities(input: $input) {
+        rules {
```

Comment:
probably want this to read back stats from queries and polices, too. they weren't originally scoped for this task, but they can be uploaded now, too.

Reply:
👍🏻  That should be pretty quick to add.

Reply:
Done in a2f1e7ed24dfb4df26963e8238c24e25c82f7ddc